examples/tutorial_server_variable_async.c

```diff
@@ -0,0 +1,241 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information. */
+
+/**
+ * Async Read Write of Variables
+ * -------------------------
+*/
+
+#include <open62541/client_config_default.h>
+#include <open62541/plugin/log_stdout.h>
+#include <open62541/server.h>
+
+#ifndef WIN32

```

Comment:
You should use `UA_ARCHITECtURE_WIN32` or `_WIN32`, but not `WIN32`. The latter works because it is defined by default by CMake, but you shouldn't rely on it.

(Note that existing code does violate this principle as well)

Reply:
Correction: since this is an example, `_WIN32` should be used. `UA_ARCHITECTURE_WIN32` is only used for core library code.

Reply:
Thank you for the hint. Will be added in the next commit.